data/sql/updates/pending_db_world/deadwoodrep.sql

```diff
@@ -0,0 +1 @@
+UPDATE `creature_onkill_reputation` SET `MaxStanding1` = 5 WHERE (`creature_id` IN (7157, 7156, 7158));
```

Comment:
A small syntax only, but the external parentheses here are unnecessary. Otherwise, this looks good!

Reply:
Sorry, it seems this is just how Keira outputs the SQL, so this is fine!